doc/release/RELEASE-NOTES.md

```diff
@@ -32,6 +32,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 *
 
 ### What's new
+* [OSDEV-1045](https://opensupplyhub.atlassian.net/browse/OSDEV-1045) - The color of the facility panel for claimed facilities in the Merge moderation screen has been changed to green. Added flag `highlightBackground` to highlight panel only on the Dashboard Merge Facilities screen.
```

Comment:
> Added flag `highlightBackground` to highlight panel only on the Dashboard Merge Facilities screen.

I think it's not relevant to the "What's new" section.

Also, please describe the changes in the FacilityIndexDetailsSerializer.